src/macaron/output_reporter/results.py

```diff
@@ -95,6 +97,8 @@ class Record(Generic[RecordNode]):
     description: str
     pre_config: Configuration
     status: SCMStatus
+    policies_passed: list
```

Comment:
The code documentation for these two attributes do not exist (if those attributes are final).

Reply:
Fixed https://github.com/oracle-samples/macaron/pull/46/commits/4e61112595339f7871796e707f604cc1ff1b5711